docs/manuals/source/TasksAndConcepts/Plugins/StorageDaemonPlugins/ScsicryptoSd.rst.inc

```diff
@@ -81,40 +81,159 @@ The following additional security is needed for the following operating systems:
 Linux (SG_IO ioctl interface):
 
 
-The user running the storage daemon needs the following additional capabilities: :index:`\ <single: Platform; Linux; Privileges>`\
+.. index::
+  single: Platform; Linux; Privileges
+
+To perform the operations required for **scsicrypto**, the programs must either run as user **root** or the additional capability :strong:`CAP_SYS_RAWIO+EP` (see capabilities(7)) must be set.
+The |sd| normally runs as user **bareos**. Running it as **root** is not recommended.
+
+If :command:`bareos-sd` does not have the appropriate capabilities, all other tape operations may still work correctly, but you will get "Unable to perform SG\_IO ioctl" errors.
+
+**systemd** (not recommended)
+
+
+To add the capabilities to bareos-sd.service you can add in file `/etc/systemd/system/bareos-sd.d/override.conf` a section containing the :strong:`AmbientCapabilities=CAP_SYS_RAWIO` line.
+The easiest way to create this file is to use the following instructions as root.
+
+.. code-block:: shell-session
+
+   systemctl edit bareos-sd.service
+
+Fill the file with the following content, then save and exit
+
+.. code-block::
```

Comment:
```suggestion
.. code-block:: cfg
```

Reply:
Currently there is a build error because of the missing parameter.

Reply:
Badly the error doesn't occur locally. fixed thx